gradle/compile-common.gradle

```diff
@@ -1,13 +1,15 @@
 /*
- * Copyright 2016-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ * Copyright 2016-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
  */
 
-// Platform-specific configuration to compile common modules
+kotlin.sourceSets {
+    commonMain.dependencies {
+        api group: 'org.jetbrains.kotlin', name: 'kotlin-stdlib-common', version: kotlin_version
```

Comment:
It would be more consistent and thus easier to read when version is specified in the same format as on the next line:  `"org.jetbrains.kotlin:kotlin-stdlib-common:$kotlin_version"`